src/main/java/com/ericsson/ei/subscriptionhandler/SendMail.java

```diff
@@ -65,47 +64,42 @@ public void setMailSender(JavaMailSender emailSender) {
     /**
      * This method takes two arguments i.e receiver mail-id and aggregatedObject and
      * send mail to the receiver with aggregatedObject as the body.
-     * 
+     *
      * @param receiver
-     * @param aggregatedObject
+     * @param mapNotificationMessage
      */
-    public void sendMail(String receiver, String mapNotificationMessage)
-            throws MessagingException, SubscriptionValidationException {
+    public void sendMail(String receiver, String mapNotificationMessage) throws MessagingException {
         Set<String> extEmails = new HashSet<>();
         try {
             extEmails = extractEmails(receiver);
         } catch (SubscriptionValidationException e) {
             e.printStackTrace();
-            log.error(e.getMessage());
+            LOGGER.error(e.getMessage());
```

Comment:
I think its better to have Exception Message before stacktrace. Because I think most of us reads the logs from up to down in the log normally and want to read the message before stacktrace.